queries.sql

```diff
@@ -8,3 +8,54 @@ select name, escape_attempts from animals where weight_kg>10.5
 select * from animals where neutered is true
 select * from animals where name != 'Gabumon'
 select * from animals where weight_kg between 10.4  and 17.3
+
+
+begin;
+update animals set species = 'unspecified';
+select * from animals;
+
+rollback;
+select * from animals;
+
```

Comment:
- [ ] @JakariaJishan great work implementing the changes requested however you still haven't added a screenshot for this transaction